### PR TITLE
Mappings for level storage-related functionality

### PR DESCRIPTION
**Pull request by bluebear94**

### CODE REVIEW

**Comment by liach** [comment]
Note: Potential conflict with #1682

File: mappings/net/minecraft/util/registry/DynamicRegistryManager.mapping [hunk 1]
```diff
@@ -45,6 +46,10 @@ CLASS net/minecraft/class_5455 net/minecraft/util/registry/DynamicRegistryManage
 		COMMENT
 		COMMENT @throws IllegalStateException if the registry does not exist
 		ARG 1 key
+	METHOD method_31060 register (Lcom/google/common/collect/ImmutableMap$Builder;Lnet/minecraft/class_5321;Lcom/mojang/serialization/Codec;Lcom/mojang/serialization/Codec;)V
+		ARG 0 infosBuilder
+		ARG 1 reegistryRef
```
Comment (Gegy) [inline mappings/net/minecraft/util/registry/DynamicRegistryManager.mapping]:
```suggestion
		ARG 1 registryRef
```

**Comment by bluebear94** [comment]
Rebased this PR onto the 1.16.3 branch; some of the mappings in this PR are now superseded by later commits upstream.

File: mappings/net/minecraft/util/registry/DynamicRegistryManager.mapping (hunk 1 above)
Comment (bluebear94) [reply mappings/net/minecraft/util/registry/DynamicRegistryManager.mapping]: Fixed this typo (the mapping was present upstream so I just used the upstream names there).

**Comment by modmuss50** [comment]
Thanks for this PR, however it seems to be quite far behind so im going to close it. If anyone wants to get this updated please open a new PR.

**Pull request closed without merging**